react-native/debugging/analyse-bug.mo.md

```diff
@@ -0,0 +1,185 @@
+# [MO] How to analyse and react to a bug on a React-Native project ?
+
+## Owner: [Thomas Pucci](https://github.com/tpucci)
+
+## Prerequisites
+
+* 🐛 You have a bug, obviously.
+* ⚛️ It lives in a React-Native project.
+
+## Scrum preamble
+
+You just spotted a bug. What should you do ?
+
+{% hint style='info' %}
+
+For M33 People:
+
+First thing first: communicate with your Product Owner and follow this [mode of operation](http://blog.m33.network/2017/07/i-just-detected-a-bug-what-should-i-do/).
+
+{% endhint %}
+
+### Option 1: You know where it comes from and how to fix it
+
+Write your technical todo list in the next ticket which should fix it. Then estimate it and prioritize it with your PO.
+
+### Option 2: You don't know where it comes from and how to fix it
+
+Priotitize a timebox with the expected behavior
+as the user story title. Label the user story **"Bug"**. In the description, write the actual behavior.
+During your timebox follow the next steps.
+
+## Steps
+
+### 1. Try to reproduce the bug with the information you have in a DEV environment
+
+You can reproduce the bug. Great! **Go to step 2.**
+
+You can't reproduce the bug. Bummer. This is one of the hardest to fix.
+
+### 1.1 Ask for more information
+
+Ask for the person who spotted the bug all information of its environment. Such as:
+
+* app version;
+* account information (username, password, mean of payment);
+* exact mode of operation which causes the bug (list of actions and waiting times): this can includes:
+  * time and dates
+  * localisation
+  * phone orientation
+  * connectivity information
```

Comment:
Can you give a good and a bad example?

Reply:
If the user is logged in, which data is on the account, and whether you can use this account or not